test/e2e/util.go

```diff
@@ -54,6 +54,21 @@ func waitForPodRunning(c *client.Client, id string) {
 	}
 }
 
+func waitForPodNotPending(c *client.Client, id string) {
```

Comment:
Ok I'll take a look, I see waitForPodSuccess already returns a bool so I could copy it here...